applications/prompt-proto-service-latiss/values-usdfprod-prompt-processing.yaml

```diff
@@ -45,6 +45,11 @@ prompt-proto-service:
   apdb:
     url: postgresql://rubin@usdf-prompt-processing.slac.stanford.edu:5432/lsst-devl
 
+  sasquatch:
+    endpointUrl: https://usdf-rsp-dev.slac.stanford.edu/sasquatch-rest-proxy
+    namespace: lsst.prompt.prod
```

Comment:
Do we want `lsst.prompt.prod` here?

Reply:
Yes? This is the real LATISS service we run most nights. That's as "production" as we can get ATM.

Reply:
Resolved in Slack; no specific objections.